server/api/review.js

```diff
@@ -3,6 +3,11 @@ const { Review } = require('../../db')
 const Sequelize = require('sequelize')
 
 router.get('/', (req, res, next) => {
-    Review.findAll().then(reviews => res.status(200).send(reviews))
```

Comment:
I don't think we have a use case for loading all reviews. I have written routes in my latest PR so you don't have to worry about routes